openqaoa/qaoa_parameters/standardparams.py

```diff
@@ -39,7 +39,7 @@ class QAOAVariationalStandardParams(QAOAVariationalBaseParams):
     hyperparameters:
         The hyperparameters containing the register, terms, weights, and the number of layers
         ``hyperparameters = (register, terms, weights, p)``
-    parameters:
+    qaoa_circuit_params:
```

Comment:
okay, I see this is incorrect. This is probably outdated docstring from the outdated script. Let's fix this

Reply:
Yes, of course! Feel free to edit and push